Mathlib/RingTheory/Congruence/Basic.lean

```diff
@@ -67,16 +67,15 @@ section Basic
 
 variable [Add R] [Mul R] (c : RingCon R)
 
--- Porting note: upgrade to `FunLike`
 /-- A coercion from a congruence relation to its underlying binary relation. -/
-instance : FunLike (RingCon R) R (R → Prop) :=
-  { coe := fun c => c.r,
-    coe_injective' := fun x y h => by
-      rcases x with ⟨⟨x, _⟩, _⟩
-      rcases y with ⟨⟨y, _⟩, _⟩
-      congr!
-      rw [Setoid.ext_iff,(show x.Rel = y.Rel from h)]
-      simp}
+instance : FunLike (RingCon R) R (R → Prop) where
+  coe := fun c => c.r
```

Comment:
```suggestion
  coe c := c.r
```
sorry I'm late for this, but does it work?

Reply:
That wouldn't work but `coe := (·.r)` would.

Reply:
Should I create a new PR to change it?
FYI `coe := fun` is present 22 times throughout Mathlib, so your proposal could be applied en masse.

Reply:
I personally like the `(·.r)` syntax, but I have a feeling that this is not a mainstream opinion.  I think that moving identifiers to the left of `:=` is welcome, though, so that would probably be uncontroversial as a change (assuming that those are already in `where` form).

Reply:
I can do your version (moving `c` to the left). That said, this is a proposal for a new PR. Do you want me to apply the change to every `coe` in Mathlib?

Reply:
I would say yes, although unless we try to seriously enforce it, it will relapse.